addons/payment/models/payment_acquirer.py

```diff
@@ -227,6 +227,32 @@ def _get_default_payment_method_id(self):
         self.ensure_one()
         return self.env.ref('account.account_payment_method_manual_in').id
 
+    #=== ONCHANGE METHODS ===#
+
+    @api.onchange('state')
+    def _onchange_state(self):
+        """ Display a warning about the consequences of disabling an acquirer. """
+        self.ensure_one()
+
+        if self.state == 'disabled':
+            related_tokens = self.env['payment.token'].search([
+                ('acquirer_id', '=', self.id.origin)
```

Comment:
```suggestion
                ('acquirer_id', '=', self._origin.id)
```
Most usages rely on `_origin.id` instead.  Both paths work indeed, but I'd prefer if you use the most used one :).